test/ui-pytest/gui/components/authenticate_popup.py

```diff
@@ -0,0 +1,26 @@
+import allure
+
+from gui.elements.qt.button import Button
+from gui.elements.qt.text_edit import TextEdit
+from gui.elements.qt.object import QObject
+
+
+
+class AuthenticatePopup(QObject):
+
+    def __init__(self):
+        super(AuthenticatePopup, self).__init__('contextMenu_PopupItem')
+        self._password_text_edit = TextEdit('sharedPopup_Password_Input')
+        self._primary_button = Button('sharedPopup_Primary_Button')
+
+    @allure.step('Authenticate in popup')
```

Comment:
Please use business logic in steps, not what we are doing with UI but what we are doing in the current process:
Authenticate actions with password?

Reply:
Fixed